src/TamrielOMeter.lua

```diff
@@ -136,11 +147,20 @@ function TamrielOMeter:TryCalculateRootMapMeasurement(rootMapIndex)
     return measurement
 end
 
-function TamrielOMeter:CalculateMapMeasurement()
+
+function TamrielOMeter:CalculateMapMeasurement(providedMapId)
     local adapter = self.adapter
+    local mapId = providedMapId
```

Comment:
Just use mapId as the parameter name. No need to create another variable as it will already be a local anyway.

Reply:
done